user/cron-jobs.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Cron Jobs
+layout: en
+permalink: /user/cron-jobs/
+---
+
+**Please note that cron jobs are not enabled by default.
+Please ask us to unlock it for your repository:
```

Comment:
Slightly picky here, but can we change to "Please ask us to unlock this feature for your repository" as "it" does not match "cron jobs"